### PR TITLE
[HttpKernel] Fix including welcome page multiple times

### LINKED ISSUE #53686: Welcome page is shown only once in event loop based runtimes.
### Symfony version(s) affected

7.0.2

### Description

In runtimes with always loaded kernel, the welcome page (route: /) is displayed only once on the first request. All subsequent requests show a blank page.

The reason is _include_once_ here: https://github.com/symfony/symfony/blob/add24fa38e0438f1188211cce7adf418f41b04a0/src/Symfony/Component/HttpKernel/EventListener/RouterListener.php#L162

### How to reproduce

Run a freshly installed, empty symfony project in any event loop based runtime, go to / path to see the welcome page, refresh the page.

### Possible Solution

Replace _include_once_ with _include_ in RouterListener.php.
Don't use functions in the _welcome.html.php_ file, because it causes function redeclaration errors when the file is included more than once.

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by HypeMC**
| Q             | A
| ------------- | ---
| Branch?       | 7.0
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| Issues        | Fix #53686
| License       | MIT

Reverts #53239 & switches to using closures instead.

### CODE REVIEW

File: src/Symfony/Component/HttpKernel/Resources/welcome.html.php [hunk 1]
```diff
@@ -1,3 +1,118 @@
...
+    <path stroke="none" d="M0 0h24v24H0z" fill="none"></path>
+    <path d="M3 19a9 9 0 0 1 9 0a9 9 0 0 1 9 0"></path>
+    <path d="M3 6a9 9 0 0 1 9 0a9 9 0 0 1 9 0"></path>
+    <path d="M3 6l0 13"></path>
+    <path d="M12 6l0 13"></path>
+    <path d="M21 6l0 13"></path>
+</svg>
+SVG;
+
+$renderCommunityIconSvg = fn () => <<<SVG
```
Comment (nicolas-grekas) [inline src/Symfony/Component/HttpKernel/Resources/welcome.html.php]: or just strings?
```suggestion
$communityIconSvg = <<<SVG
```

Comment (HypeMC) [reply src/Symfony/Component/HttpKernel/Resources/welcome.html.php]: :man_facepalming: how did I miss something so obvious... fixed.

**Comment by nicolas-grekas** [comment]
For 5.4? Why 7.0?

**Comment by HypeMC** [comment]
@nicolas-grekas The functions were introduced in 7.0: #51771

**Review by nicolas-grekas — APPROVED** [review]

**Review by javiereguiluz — APPROVED** [review]

**Review by xabbuh — APPROVED** [review]

**Comment by nicolas-grekas** [comment]
Thank you @HypeMC.

**Pull request merged**